### ISSUE TITLE
[Feature Request] Add multi_attach option to EBS creation.

### ISSUE THREAD

**Issue by fishyfishphil** [terraform-providers/terraform-provider-aws#13104]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

EBS provisioned iops volumes can now be attached to multiple EC2 instances, the multi attach can only be enabled at creation. Adding the option to enable the multi attach would be very useful.

### New or Affected Resource(s)

* aws_ebs_volume

### Potential Terraform Configuration

```hcl
resource "aws_ebs_volume" "ebs_volume" {
    size = 10
    type = "io1"
    iops = 500
    multi_attach = "true"
}
```

### References

https://aws.amazon.com/about-aws/whats-new/2020/02/ebs-multi-attach-available-provisioned-iops-ssd-volumes/

**Comment by DrFaust92** [comment]
addressed in #13108

**Comment by anGie44** [comment]
The fix for this has been merged and will release with version 2.61.0 of the Terraform AWS Provider, expected in this week's release.
